share/picongpu/examples/FieldAbsorberTest/include/picongpu/param/pml.param

```diff
@@ -33,7 +33,7 @@ namespace fields
 {
 namespace maxwellSolver
 {
-namespace yeePML
+namespace Pml
 {
 
     /* The parameters in this file are only used if field solver is YeePML.
```

Comment:
Perhaps you want to update this line as well, to be consistent with other changes in the comments?

Reply:
Actually not for the moment. As `YeePML` ist the only field solver that uses PMLs currently. I am just afraid that I/we will forget to change this line once there is another field solver making use of PMLs.